.github/workflows/github-pages.yml

```
@@ -0,0 +1,29 @@
+# GitHub Action for Jekyll
+#
+# Runs jekyll build on master and pushes output to gh-pages branch.
+# (Then GitHub deploys that static content.)
+
+name: Build & Deploy Jekyll to GitHub Pages
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  github-pages:
+    runs-on: ubuntu-latest
+    container: ruby:2.7.1-buster
```

```suggestion
    container: ruby:2-buster
```

(Just so we don't have to update minor versions frequently.)